introduction.md

```diff
@@ -50,6 +50,26 @@ Below are the type of questions phinetics works on.
 
 * Are computers and robots morally responsible for their actions? 
 
+### Autonomous Machines
+
+* Can software possess autonomy?
```

Comment:
"autonomy" is ambivalent. Do you mean:
* a computer's _right_ to control itself? 
* whether a computer can manage itself without the assistance or intervention of a human? 
* Kantian autonomy, i.e. 

> the capacity of an agent to act in accordance with objective morality rather than under the influence of desires.